canonical-web-and-design/microk8s.io
# Fix broken JS install tabs

## Done
- Updated the js paths
- Fixed some small bugs in the scripts
- Removed the built js from source 

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Check the network tab as no 404
- Check that there are no js errors in the console
- Check the "Get started with Microk8s" section works as expected/

## Issue / Card
Fixes https://github.com/canonical-web-and-design/microk8s.io/issues/274
